scripts/ampHtmlValidator/checkManifest/index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+import { JSDOM } from 'jsdom';
+
+const getManifestFile = async url => {
+  const response = await fetch(url);
+
+  const html = await response.text();
+
+  const {
+    window: { document },
+  } = new JSDOM(html);
+
+  return document.querySelector('link[rel="manifest"]').getAttribute('href');
+};
+
+export default async () => {
+  const publicServiceUrls = [
+    '/news/articles/c0eg99qjynvo.amp',
+    '/sport/tennis/articles/cedlgl4lj23o.amp',
+    '/newsround/articles/cp8v6lm0ek6o.amp',
+    '/cymrufyw/erthyglau/c4ge78ry9dmo.amp',
+    '/naidheachdan/sgeulachdan/c3w14wqg1x8o.amp',
+  ];
+
+  const testResults = await Promise.all(
+    publicServiceUrls.map(async url => {
+      const [, service] = url.split('/');
+      const liveManifestFile = await getManifestFile(
```

Comment:
Would it be simpler to just point the urls to the mainfest files to begin with instead of fetching? i.e /news/articles/manifest.json? Or is this safer in case PS change which manifest files the urls should point to.. ?

Reply:
If you have a read of the Slack thread linked in the attached ticket it might make more sense. Otherwise, happy to jump on a quick call to explain!